packages/SwingSet/src/supervisors/subprocess-xsnap/supervisor-subprocess-xsnap.js

```diff
@@ -342,4 +342,10 @@ function makeWorker(port) {
 // xsnap provides issueCommand global
 const port = managerPort(globalThis.issueCommand);
 const worker = makeWorker(port);
+
+// Send unexpected console messages to the manager port.
+globalThis.print = (...args) => {
```

Comment:
This makes my head hurt a little. I defer to @warner

Reply:
I prefer that somebody else reviews the whole PR.

I'm not confident that I'm in sync with the discussion since your [March 5 comment](https://github.com/Agoric/agoric-sdk/issues/4737#issuecomment-1059834032):
> ISTR @erights or @kriskowal had a sketch for what ideal unhandled promise rejection tracking would look like. Can we suggest that to the XS folk?

Sending `globalThis.print` to the manager port specifically seems like something where @warner is quite a bit more swapped in than I am.

I'm open to a 15-ish minute call to get swapped in on all of the above.

Reply:
I’ll also point out that `p.catch(() => {})` defeats Node.js’s trick for revealing most stray promise chains, leaving you more in need of a causal debugger.

Reply:
> If all that is true, then this PR shifts an extra chunk of logging from the worker's stdout to (the kernel port and then to both the kernel process' stdout and any other special destination), but most of the previous logging was already following that path, and this is just to grab the low-level unhandled-rejection messages coming out of the engine. 

That's all correct.

> That sounds fine to me.

Great, thanks!  Would you be able to approve accordingly?  I'm marking as automerge.

Reply:
it landed already, but yep, given that answer, I'm r+